### PR TITLE
update overview layout

### PR DESCRIPTION
**Pull request by Lijiaoa**
TODO
- [x] table滚动条，固定表头
~固定一屏幕~

related issue: #2937

image

init:
image

### CODE REVIEW

**Comment by Lijiaoa** [comment]
TODO:
image

File: ts/webui/src/components/Overview.tsx [hunk 1]
```diff
@@ -167,6 +166,13 @@ class Overview extends React.Component<{}, OverviewState> {
                                             </Stack>
                                         </div>
                                     </Stack>
+                                    <div className='overviewChart'>
+                                        <Accuracy
+                                            accuracyData={accuracyGraphData}
+                                            accNodata={noDataMessage}
+                                            height={290}
```
Comment (colorjam) [inline ts/webui/src/components/Overview.tsx:173]: Is it possible to move the styles into the css file?

File: ts/webui/src/components/overview/experiment/BasicInfo.tsx [hunk 2]
```diff
@@ -23,76 +23,74 @@ export const ReBasicInfo = (): any => {
...
+                                        onDismiss={onDismiss}
+                                        setInitialFocus={true}
+                                    >
+                                        <div className={styles.header}>
+                                            <p className={styles.title} id={labelId} style={{ color: '#333' }}>
+                                                Error
+                                            </p>
+                                        </div>
+                                        <div className={styles.inner}>
+                                            <p className={styles.subtext} id={descriptionId} style={{ color: '#333' }}>
```
Comment (colorjam) [inline ts/webui/src/components/overview/experiment/BasicInfo.tsx:62]: move the color to style file?

Comment (Lijiaoa) [reply ts/webui/src/components/overview/experiment/BasicInfo.tsx:62]: ok

File: ts/webui/src/components/Overview.tsx (hunk 1 above)
Comment (Lijiaoa) [reply ts/webui/src/components/Overview.tsx:173]: ok

**Review by ultmaster — APPROVED** [review]

**Review by colorjam — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**